README.md

```diff
@@ -247,6 +247,7 @@ The following resources include cisco types and providers along with cisco provi
 * VXLAN Types
   * [`cisco_vxlan_global`](#type-cisco_vxlan_global)
   * [`cisco_vxlan_vtep`](#type-cisco_vxlan_vtep)
+  * [`cisco_vxlan_vtep_vni`](#type-cisco_vxlan_vtep_vni)
```

Comment:
Add this to to (by Name) section as well:

# Cisco Resource Type Catalog (by Name)